src/components/Table/Table.tsx

```diff
@@ -164,6 +164,7 @@ export const Table = ({
             <tr
               className="govuk-table__row"
               data-test="header-row"
+              tabIndex={0}
```

Comment:
Is there a reason we don't use a literal here eg. `"0"`? :smile:

Reply:
ahh not sure... you think it should be  tableIndex = '0' ?